src/AppInstallerCLICore/Workflows/SourceFlow.cpp

```diff
@@ -272,6 +272,39 @@ namespace AppInstaller::CLI::Workflow
         }
     }
 
+    void EditSources(Execution::Context& context)
+    {
+        // We are assuming there is only one match, as SourceName is a required parameter.
+        const std::vector<Repository::SourceDetails>& sources = context.Get<Data::SourceList>();
+        bool isExplicit = context.Args.Contains(Args::Type::SourceExplicit);
+        for (const auto& sd : sources)
+        {
+            // Get the current source with this name.
+            Repository::Source currentSource{ sd.Name };
+
+            // Only permitting Explicit to be edited at this time.
+            Repository::Source source{ sd.Name, isExplicit };
+
+            // Is anything being changed??
+            if (!source.IsEditOfSource(currentSource))
+            {
+                context.Reporter.Info() << Resource::String::SourceEditNoChanges(Utility::LocIndView{ sd.Name }) << std::endl;
+                continue;
+            }
+
+            context.Reporter.Info() << Resource::String::SourceEditOne(Utility::LocIndView{ sd.Name }, Utility::LocIndView{ Utility::ConvertBoolToString(isExplicit) }) << std::endl;

```

Comment:
This is likely to become unusable if edit expands to other properties.  My first thought is to use a table with old and new properties, similar to `winget source list winget`:
```
Field       Value                                  New Value
----------------------------------------------------------------
Name        winget
Type        Microsoft.PreIndexed.Package
Argument    https://cdn.winget.microsoft.com/cache
Data        Microsoft.Winget.Source_8wekyb3d8bbwe
Identifier  Microsoft.Winget.Source_8wekyb3d8bbwe
Trust Level Trusted|StoreOrigin
Explicit    false                                  true
Updated     2025-12-01 16:15:11.000
```
You could of course remove some uneditable fields like the update time (or only have the currently editable fields listed at all).

Reply:
Updated to list the changed values in this manner.